db/migrate/20161107015702_invoice_data_to_jso_nb.rb

```diff
@@ -0,0 +1,6 @@
+class InvoiceDataToJsoNb < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :invoices, :data
+    add_column :invoices, :data, :jsonb, null: false, default: '{}'
```

Comment:
Clean up these migrations. Really you should have just changed this one in place without adding the second one to clean it up. Also, not an issue for us, but consider if we already had this deployed to production. Removing the column would delete any data we would have had there. I only mention this because I already had an invoice stored locally and this migration blew that away. Just something to consider.